source/meshconv/CMakeLists.txt

```
@@ -0,0 +1,48 @@
+cmake_minimum_required(VERSION 3.16 FATAL_ERROR)
+set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+project (meshconv CXX)
+
+IF (NOT MR_EMSCRIPTEN)
+  find_package(Boost COMPONENTS program_options REQUIRED )
+  if(Boost_PROGRAM_OPTIONS_FOUND)
+    add_definitions( "-DCGAL_USE_BOOST_PROGRAM_OPTIONS" )
+    link_libraries( ${Boost_PROGRAM_OPTIONS_LIBRARY} )
```

```suggestion
  find_package(Boost COMPONENTS program_options REQUIRED )
```

Sorry, I've just checked, and there are some extra lines

---

And you will need to add `${BOOST_PROGRAM_OPTIONS_LIBRARY}` to `target_link_library()`